src/java.base/share/classes/java/lang/VirtualThread.java

```diff
@@ -417,8 +421,6 @@ private void run(Runnable task) {
                 }
 
             } finally {
```

Comment:
This means an empty finally block, I assume you'll remove the try-finally here.

Reply:
Thank you for the comment. I can move the try-finally to the method `java/lang/VirtualThread$VThreadContinuation$1.run()` if you prefer. But it will play the same role functionally.

Reply:
Changing the run method to
```
vthread.notifyJvmtiStart();
try {
    vthread.run(task);
} finally {
    vthread.notifyJvmtiEnd();
}
```
is okay. I was initially puzzled because the run method is already`@Hidden` but I don't think this is used by JVMTI.

Reply:
Thanks. Yes, the annotation `@Hidden` is not used by the JVMTI. It would be tricky to use it consistently.